packages/manager/modules/pci/src/components/users/download-rclone/download-rclone.controller.js

```diff
@@ -38,7 +38,12 @@ export default class PciUsersDownloadRcloneController {
       id: region,
       label: this.ovhManagerRegionService.getTranslatedMicroRegion(region),
     }));
+    this.storageS3Regions = map(this.storageS3Regions, (region) => ({
+      id: region,
+      label: this.ovhManagerRegionService.getTranslatedMicroRegion(region),
+    }));
```

Comment:
**suggestion:** `Array.prototype.map` can be used.

Reply:
Okay Anoop